openapi/components/schemas/Cashier/CashierCustomPropertySet.yaml

```
@@ -0,0 +1,30 @@
+type: object
+required:
+  - name
+  - properties
+
+properties:
+  id:
+    readOnly: true
+    allOf:
+      - $ref: ../ResourceId.yaml
+  name:
+    type: string
+    description: Name of the custom property set.
+  properties:
+    description: |-
+      Defines properties the user can complete when they use the hosted cashier form.
+      This field accepts [JSON-schema](https://json-schema.org/) drafts 4, 6, and 7.
+      Accepted properties types are: `string`, `number`, `integer`, `array` and `enum`.
+      For more information on how to set up the fields look at [documentation](https://www.rebilly.com/docs/settings/cashier-strategies/).
```

nitpick: [serial comma](https://product.rebilly.dev/writing-style-guide/grammar-and-mechanics/commas/#serial-commas) and link format
```suggestion
      Accepted properties types are: `string`, `number`, `integer`, `array`, and `enum`.
      For more information, see [Cashier strategies](https://www.rebilly.com/docs/settings/cashier-strategies/).
```